src/client/components/vision2/camera/stories/greenhorizon.stories.tsx

```diff
@@ -0,0 +1,105 @@
+import { storiesOf } from '@storybook/react'
+import { action } from 'mobx'
+import { computed } from 'mobx'
+import { observable } from 'mobx'
+import { reaction } from 'mobx'
+import { disposeOnUnmount } from 'mobx-react'
+import { now } from 'mobx-utils'
+import { Component } from 'react'
+import React from 'react'
+import * as THREE from 'three'
+
+import { range } from '../../../../../shared/base/range'
+import { Matrix4 } from '../../../../math/matrix4'
+import { Vector2 } from '../../../../math/vector2'
+import { Vector3 } from '../../../../math/vector3'
+import { fullscreen } from '../../../storybook/fullscreen'
+import { scene } from '../../../three/builders'
+import { orthographicCamera } from '../../../three/builders'
+import { stage } from '../../../three/builders'
+import { Canvas } from '../../../three/three'
+import { Three } from '../../../three/three'
+import { GreenHorizonViewModel } from '../greenhorizon'
+import { Lens } from '../model'
+import { Projection } from '../model'
+import { GreenHorizon } from '../model'
+
+storiesOf('components.vision2.camera.greenhorizon', module)
```

Comment:
Were you going to have the story of the actual image as well?

Reply:
#313 😄